(Font) Added get_message_width in font_renderer_t

natinusala opened the pull request:
I thought that a menu driver could need to get a message on-screen width, for example to draw several strings horizontally on a same line. It just has to include `font_renderer_driver.h` to use the function.

I added the function in `font_renderer_t` and only implemented it in `gl_raster_font` (it's set to `NULL` on the others, and so returns 0 when used). The function in `font_renderer_driver.h` is only there to create a shortcut so we can use it more directly, like we have with other drivers.

I don't know if I placed the stuff at the right place tho, I tried to match as much as possible with how the drivers works, but it looks right to me.

twinaphex commented:
OK, does it default to the current default behavior when this function callback isn't bound/implemented in a font renderer driver?

natinusala commented:
I added it to all the current drivers, for now only gl_raster has a working one (the others have NULL). I added a check in `font_renderer_get_message_width`, if it is not implemented (NULL) then it will return 0. 

Currently the function isn't used by anyone but gl_raster to compute alignment (and I didn't touch it), so the default behavior is not modified.

natinusala commented:
And I did nothing but adding the function in the driver, so it's not altering anything else

twinaphex commented:
OK, sounds good then.
